cmd/stack_yaml_lint.go

```diff
@@ -139,3 +142,26 @@ func (stackDetails *StackYaml) checkLicense(log *LoggingConfig) int {
 	}
 	return stackLintErrorCount
 }
+
+func (stackDetails *StackYaml) checkRequirements(log *LoggingConfig) int {
+	stackLintErrorCount := 0
+
+	reqsMap := map[string]string{
+		"Docker":  stackDetails.Requirements.Docker,
```

Comment:
What happens when `requirements` is not specified in `stack.yaml`? I believe this like will panic. Please test this and fix accordingly.

Reply:
If requirements aren't set in the `stack.yaml`, the linter will continue. Just tested this now. I believe the values are set as empty strings if the requirement isn't set hence why I added the `if req == ""; continue` lines.

Reply:
OK, I was worried `stackDetails.Requirements` would be nil, but I suppose since it is part of the StackYaml struct it gets a value. 

Thanks for testing. I'll approve and merge.